benches/benches/block_target_gas_set/memory.rs

```diff
@@ -0,0 +1,277 @@
+        group,
+        vec![
+            op::cfei(100),
+            op::cfsi(10),
+            op::cfsi(10),
+            op::cfsi(10),
+            op::cfsi(10),
+            op::cfsi(10),
+            op::cfsi(10),
+            op::cfsi(10),
+            op::cfsi(10),
+            op::cfsi(10),
+            op::cfsi(10),
+            op::jmpb(RegId::ZERO, 10),
+        ],
+        vec![],
+    );
+
+    run(
+        "memory/lb opcode",
+        group,
+        [op::lb(0x10, RegId::ONE, 10), op::jmpb(RegId::ZERO, 0)].to_vec(),
+        vec![],
+    );
+
+    run(
+        "memory/lw opcode",
+        group,
+        [op::lw(0x10, RegId::ONE, 10), op::jmpb(RegId::ZERO, 0)].to_vec(),
+        vec![],
+    );
+
+    for i in arb_dependent_cost_values() {
+        let id = format!("memory/mcl opcode {:?}", i);
+        run(
+            &id,
+            group,
+            vec![
+                op::movi(0x11, i),
+                op::aloc(0x11),
+                op::move_(0x10, RegId::HP),
+                op::mcl(0x10, 0x11),
+                op::jmpb(RegId::ZERO, 0),
+            ],
+            vec![],
+        );
+    }
+
+    for i in arb_dependent_cost_values() {
+        let id = format!("memory/mcli opcode {:?}", i);
+        run(
+            &id,
+            group,
+            vec![
+                op::movi(0x11, i),
+                op::aloc(0x11),
+                op::move_(0x10, RegId::HP),
+                op::mcli(0x10, i),
+                op::jmpb(RegId::ZERO, 0),
+            ],
+            vec![],
+        );
+    }
+
+    for i in arb_dependent_cost_values() {
+        let id = format!("memory/mcp opcode {:?}", i);
+        run(
+            &id,
+            group,
+            vec![
+                op::movi(0x11, i),
+                op::aloc(0x11),
+                op::move_(0x10, RegId::HP),
+                op::mcp(0x10, RegId::ZERO, 0x11),
+                op::jmpb(RegId::ZERO, 0),
+            ],
+            vec![],
+        );
+    }
+
+    let valid_values: Vec<_> = arb_dependent_cost_values()
+        .iter()
+        .copied()
+        .take_while(|p| *p < (1 << 12)) // 12 bits
+        .collect();
+    for val in valid_values {
+        let id = format!("memory/mcpi opcode {:?}", val);
+        let val_as_u16 = (val).try_into().unwrap();
+        run(
+            &id,
+            group,
+            vec![
+                op::movi(0x11, val),
+                op::aloc(0x11),
+                op::move_(0x10, RegId::HP),
+                op::mcpi(0x10, RegId::ZERO, val_as_u16),
+                op::jmpb(RegId::ZERO, 0),
+            ],
+            vec![],
+        );
+    }
+
+    for i in arb_dependent_cost_values() {
+        let id = format!("memory/meq opcode {:?}", i);
+        let mut script = vec![op::move_(0x11, RegId::ZERO), op::move_(0x12, RegId::ZERO)];
+        script.extend(set_full_word(0x13, i as u64));
+        script.extend(vec![
+            op::meq(0x10, 0x11, 0x12, 0x13),
```

Comment:
```suggestion
            op::meq(0x10, RegId::ZERO, RegId::ZERO, 0x13),
```

You can use `RegId::ZERO` directly=)